ReSharper.FSharp/test/data/features/quickFixes/addIgnore/Availability - If expression.fs

```diff
@@ -0,0 +1,7 @@
+module Test
+
+type A() =
```

Comment:
What does type declaration add to this test?

Reply:
Simplified